docs/tutorial/snapcraft.md

```diff
@@ -97,7 +96,7 @@ grade: stable
 
 apps:
   electron-packager-hello-world:
-    command: electron-quick-start/electron-quick-start --no-sandbox
+    command: minimal-repro/minimal-repro --no-sandbox
```

Comment:
I'd suggest using a different project here. We'd have to think about which project would be a good fit.